OMG.Classroom/ApplicationLayer/Services/AssignmentService.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using DataAccessLayer.Entities;
+using DataAccessLayer.Repositories.AssignmentRepositories;
+using Microsoft.EntityFrameworkCore;
+
+namespace ApplicationLayer.Services
+{
+    public class AssignmentService : IAssignmentService
+    {
+        private readonly IAssignmentRepository _assignmentRepository;
+
+        public AssignmentService(IAssignmentRepository assignmentRepository)
+        {
+            _assignmentRepository = assignmentRepository;
+        }
+        public async Task CreateAsync(Assignment assignment)
```

Comment:
You should use dtos instead of entites. The idea behind it is that we don't want to transfer all the data since part of it might be unnecessary (for example - we don't need to pass an id to the create method)